052-SAPAppModernization/Student/Challenge-08.md

```diff
@@ -0,0 +1,26 @@
+# Challenge 08 - Azure integration, Logic Apps and EAI
+
+[< Previous Challenge](./Challenge-07.md) - **[Home](../README.md)** 
+
+## Introduction
+At this point we will talk about other methods of streaming and sending data to and from SAP systems using Azure as an integration 'Broker'.
+
+## Description
+We can use SAP's gateway services and Logic Apps to directly call existing code from Azure services in a convenient manner.
+
+Install a new vm into your vnet or use an existing already running Windows VM and install the Logic Apps On-premises data gateway [instructions here](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-gateway-install)
```

Comment:
Capitalize "vm" to "VM"

Reply:
Since it's the first time in this challenge we mention it, I've de-acronymed it to 'Virtual Machine'